### ISSUE TITLE
Serious Sam installation error (file path mismatch)

### ISSUE THREAD

**Issue by Rak1ta** [jmcerrejon/PiKISS#153]
In game shortcuts I have paths like this:
```
/home/pi/games/ssam-tfe/Bin/ssam-tfe
/home/pi/games/ssam-tse/ssam-tse.sh
```
But the game has different paths and names of executable files (see screenshot).

Where should [these files](https://github.com/jmcerrejon/PiKISS/blob/084c8eb1491124b1efd25a34384d7429735aaeb2/scripts/games/ssam.sh#L118) come from?

i use: RaspiOS 32bit Debian10 Buster/11 Bullseye

![Безымянный](https://user-images.githubusercontent.com/88277343/175397938-894f3469-c44b-41cd-9559-5cde626b53c8.png)

**Comment by jmcerrejon** [comment]
If you follow the guide at the end of the installation ([here](https://github.com/ptitSeb/Serious-Engine#copy-official-game-data-optional) is the link), you can read the files goes to /home/pi/games/ssam/ssam-tfe/Bin folder or /home/pi/games/ssam/ssam-tse/Bin. That's all 😄

So copy all the game data files from Steam (or whatever) inside the correct folder. 

Anyway, I'm going to change some files in the installation script to make it easier. 😄

**Comment by jmcerrejon** [comment]
Fixed 97d0fe785d2d1ff5c6ffc7bbf4b86c1400756dc9
